src/shared/utils/reflect.ts

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import type { ReflectServerOptions } from "@rocicorp/reflect/server"
+import { WriteTransaction } from "@rocicorp/reflect"
+import { generate } from "@rocicorp/rails"
+import { Reflect } from "@rocicorp/reflect/client"
+import { validateReflectClientState } from "shared/utils/validators"
+import { ReflectCursor, ReflectUserInfo } from "shared/types"
+import { nanoid } from "@reduxjs/toolkit"
+
+function getParse<T>(validator: (val: any) => T) {
+  return process.env.NODE_ENV !== "production" ? validator : (val: T) => val
+}
+
+export const {
+  init: initClientState,
+  get: getClientState,
+  update: updateClientState,
+  delete: deleteClientState,
+  list: listClientState,
+} = generate("client-state", getParse(validateReflectClientState))
```

Comment:
Can you add a comment what is going on here with `getParse`? Why do we need a different thing for production?

Reply:
It was needed for type safety, but seems like it is not really necessary to have it